async_storage/LocalStorage.ts

```diff
@@ -0,0 +1,36 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+const FAVOURITE_KEY = "Favourite"
+
+export async function getFavourites(): Promise<Array<number>> {
+    try {
+        const raw = await AsyncStorage.getItem(FAVOURITE_KEY);
+        if (raw?.trim().length == 0 ?? true) return [];
+        return <number[]>JSON.parse(raw!);
```

Comment:
I would simplify this code
```suggestion
        const value = await AsyncStorage.getItem(FAVOURITE_KEY);
        return value ? JSON.parse(raw) as Array<number> : []
```